cfme/web_ui/listaccordion.py

```diff
@@ -104,7 +104,6 @@ def select(name, link_title_or_text, by_title=True, partial=False):
     """
     return _get_link(name, link_title_or_text, by_title, partial).click()
 
-
```

Comment:
Isn't this a lint error? - Thought we always needed two blank spaces at module level